### PR TITLE
added notes about supported versions

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#6075

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 1]
```diff
@@ -65,10 +65,10 @@ You can set several rule options within a WDAC policy. Table 1 describes each ru
...
-| **19 Enabled:Dynamic Code Security** | Enables policy enforcement for .NET applications and dynamically-loaded libraries. |
+| **16 Enabled:Update Policy No Reboot** | Use this option to allow future WDAC policy updates to apply without requiring a system reboot. NOTE: This option is only supported with the Windows 10 version 1709 and higher.|
+| **17 Enabled:Allow Supplemental Policies** | Use this option on a base policy to allow supplemental policies to expand it. NOTE: This option is only supported with the Windows 10 version 1903 and higher. |
+| **18 Disabled:Runtime FilePath Rule Protection** | Disable default FilePath rule protection (apps and executables allowed based on file path rules must come from a file path that’s only writable by an administrator) for any FileRule that allows a file based on FilePath. NOTE: This option is only supported with the Windows 10 version 1903 and higher. |
+| **19 Enabled:Dynamic Code Security** | Enables policy enforcement for .NET applications and dynamically-loaded libraries. NOTE: This option is only supported with the Windows 10 version 1803 and higher. |
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md]: This suggestion is based on recommendations from other copy reviews in previous PRs. You may choose to alter some of my changes, if needed.
```suggestion
| **16 Enabled:Update Policy No Reboot** | Use this option to allow future WDAC policy updates to apply without requiring a system reboot. NOTE: This option is only supported on Windows 10, version 1709, and above.|
| **17 Enabled:Allow Supplemental Policies** | Use this option on a base policy to allow supplemental policies to expand it. NOTE: This option is only supported on Windows 10, version 1903, and above. |
| **18 Disabled:Runtime FilePath Rule Protection** | Disable default FilePath rule protection (apps and executables allowed based on file path rules must come from a file path that’s only writable by an administrator) for any FileRule that allows a file based on FilePath. NOTE: This option is only supported on Windows 10, version 1903, and above. |
| **19 Enabled:Dynamic Code Security** | Enables policy enforcement for .NET applications and dynamically-loaded libraries. NOTE: This option is only supported on Windows 10, version 1803, and above. |
```

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@isbrahm @Dansimp - PR has been copy edited and is ready for final review. This is based on issue #6075.

Thank you.

cc: @kenwith

**Comment by illfated** [comment]
Looks like this PR is ready for final review by the manager.

**Comment by mypil** [comment]
@Dansimp @kenwith - Can you possibly assist to merge this PR? This has been approved by the author jsuther1974. 

Thank you.

**Pull request merged**